.github/workflows/test-hubapp-dind.yml

```diff
@@ -0,0 +1,44 @@
+name: Test hubapp with hubpods
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  hubapp-hubpod-test:
+    if: |
+      !startsWith(github.event.head_commit.message, 'chore') &&
+      !startsWith(github.event.head_commit.message, 'build: hotfix') &&
+      !endsWith(github.event.head_commit.message, 'reformatted by jina-dev-bot')
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.7]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Build jina test-pip
+        run: |
+          docker build -f Dockerfiles/pip.Dockerfile -t jinaai/jina:test-pip .
+          docker build --build-arg PIP_TAG="[devel]" -f Dockerfiles/pip.Dockerfile -t jinaai/jina:test-pip .
```

Comment:
Is it a typo here? I can't see the reason for building the docker image twice with different pip tags and the same image tag.

Reply:
I had the same question, just copied from our `CI.yml` actually!

Reply:
Try to remove in both places